trax/models/research/funnel_transformer_test.py

```diff
@@ -0,0 +1,83 @@
+# coding=utf-8
+# Copyright 2020 The Trax Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+"""Tests for Funnel-Transformer models."""
+
+import numpy as np
+from absl.testing import absltest
+from absl.testing import parameterized
+
+from trax import layers as tl, shapes
+from trax.models.research.funnel_transformer import PoolLayer, \
+  _FunnelResidualBlock, \
+  FunnelTransformerEncoder, \
+  FunnelTransformer
+
+
+class FunnelTransformerTest(parameterized.TestCase):
```

Comment:
Can we change all tests to use, approximately, the smallest possible models? E.g. d_model=8, d_ff=8, n_layers=2 etc. This would speed up tests very significantly while testing the same functionality.

Reply:
I have changed these parameters in tests, now they run much faster.